src/Sarif.UnitTests/SarifVersionOneToCurrentTests.cs

```diff
@@ -1314,12 +1314,22 @@ public void SarifTransformerTests_ToCurrent_TwoResultsWithFixes()
                   },
                   ""replacements"": [
                     {
-                      ""offset"": 720,
-                      ""insertedBytes"": ""Jw==""
+                      ""deletedRegion"": {
+                        ""offset"": 720,
+                        ""length"": 1
```

Comment:
Given the contents of the v1 `replacement` object, the length of the region is 0. The region expresses an "insertion point".

Reply:
... and yet the test is passing, so something else must be wrong (presumably it would fail if you changed this "1" to the (correct) value of 0).

Reply:
Fixed